<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Type:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProductType">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NonPerishable</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Perishable</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Food</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Chemical</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Raw Materials</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Supplier:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSupplier">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditProductName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelExpirationDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Expiration Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditExpiration">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>440</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
  </widget>
  <widget class="QLabel" name="labelStorageCondition">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>400</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Storage Condition:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditStorageCondition">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Price:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrice">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxQuantity">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAddProduct">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Product</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidgetProducts">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDeleteProduct">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Selected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonViewAll">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>View All Products</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonBuyProduct">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Buy Product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSellProduct">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Sell Product</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCheckLowStock">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Low Stock</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonViewSuppliers">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>230</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>View Suppliers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonGenerateReport">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Report</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Product ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditProductId">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelQuantity">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBuySellQuantity">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>110</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLogout">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonChangePassword">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSaveInventory">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Inventory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonLoadInventory">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>330</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Inventory</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
